html/views.py

```diff
@@ -503,6 +507,114 @@ def download_installers_page(request, build_id):
 
 
 
+def fastlane_page(request):
+  """
+  <Purpose>
+    Renders a key and installer download page for a
+    default user-built seattle build, i.e.:
+      One 80 per-cent vessel, one owner/user
+
+      Note: 
+      owner/user-name can be set in settings.FASTLANE_USER_NAME
+  <Arguments>
+    request:
+      A Django request.
+  <Exceptions>
+    None.
+  <Side Effects>
+    If new session
+    - Creates and stores vesselinfo to appropriate location on disk
+    - Stores generated key pair to session (memory)
+  <Returns>
+    A Django response.
+  """
+
+  try:
+    build_result = False
+
+    # We have to check if the user already has a build result in his session
+    # and make sure it's a fastlane build, i.e.
+    # it does not collide with a build from the interactive CIB
+    if 'build_results' in request.session.keys():
+      for val in request.session['build_results'].values():
+        if isinstance(val, dict) and val.get("fast_lane_build"):
+          build_result = val
+          break
+    else:
+      # This dict will only be saved if the build succeeds
+      request.session['build_result'] = {}
```

Comment:
Thanks for spotting that. We really should start writing unit tests for django Seattle components too.